internal/storage/internal/tests.go

```diff
@@ -18,9 +18,12 @@ import (
 // MutateStoreFn points to a function which mutates the store (ex: add, delete a policy).
 type MutateStoreFn func() error
 
-func TestSuiteReloadable(store storage.ReloadableStore, addFn, deleteFn MutateStoreFn) func(*testing.T) {
+func TestSuiteReloadable(store storage.Store, addFn, deleteFn MutateStoreFn) func(*testing.T) {
```

Comment:
I'm wondering if the first parameter can be of type `storage.Reloadable`. The function seems to be used in storage tests which instantiate structs implementing `storage.Reloadable`.